openml-lightgbm/README.md

```diff
@@ -0,0 +1,30 @@
+# Microsoft LightGBM Provider
```

Comment:
Should we add a quick section on how does the implementation work? (i.e. this is not a "real" implementation, but rather a delegate to the the Microsoft C++ implementation)

Reply:
Maybe saying it is a "java wrapper" for LightGBM is enough?

Reply:
Fine by me 👍

Reply:
Fine by me! 👍